client/src/main.rs

```diff
@@ -93,84 +110,102 @@ async fn main() {
+	}
+
+	if new_data.is_none() {
+		return Err("Please provide the update data, e.g. update score \"Alice 100\"".to_string());
+	}
+
+	let data = new_data.unwrap();
 
-			if let Some(as_address) = as_address {
-				let as_address_parsed: Result<Address, _> = as_address.parse();
-				if as_address_parsed.is_err() {
-					eprintln!("Failed to parse address!");
-					return;
-				}
+	match Config::from_str(&field.unwrap())? {
+		Config::AttestationStationAddress => {
+			let as_address_parsed: Result<Address, _> = data.parse();
 
-				client_config_updated.as_address = as_address;
+			match as_address_parsed {
+				Ok(_) => config.as_address = data,
+				Err(_) => return Err("Failed to parse address.".to_string()),
 			}
+		},
+		Config::Mnemonic => match Mnemonic::<English>::new_from_phrase(&data) {
+			Ok(_) => config.mnemonic = data,
+			Err(_) => return Err("Failed to parse mnemonic.".to_string()),
+		},
+		Config::NodeUrl => match Http::from_str(&data) {
+			Ok(_) => config.ethereum_node_url = data,
+			Err(_) => return Err("Failed to parse node url.".to_string()),
+		},
+		Config::Score => {
+			let input: Vec<String> = data.split(" ").map(|x| x.to_string()).collect();
 
-			if let Some(mnemonic) = mnemonic {
-				let parsed_mnemonic = Mnemonic::<English>::new_from_phrase(&mnemonic);
-				if parsed_mnemonic.is_err() {
-					eprintln!("Failed to parse mnemonic!");
-					return;
-				}
-				client_config_updated.mnemonic = mnemonic;
+			if input.len() != 2 {
+				return Err("Invalid input format. Expected: \"Alice 100\"".to_string());
 			}
 
-			if let Some(node_url) = node_url {
-				let provider = Http::from_str(&node_url);
-				if provider.is_err() {
-					eprintln!("Failed to parse node url!");
-					return;
-				}
-				client_config_updated.ethereum_node_url = node_url;
+			let name = input[0].clone();
+			let score = input[1].clone();
+
+			let score_parsed: Result<u128, _> = score.parse();
+			if score_parsed.is_err() {
+				return Err("Failed to parse score.".to_string());
 			}
 
-			let res = write_json_data(client_config_updated, "client-config");
-			if res.is_err() {
-				println!("Failed to same updated config!");
+			let available_names: Vec<String> =
+				user_secrets_raw.iter().map(|x| x[0].clone()).collect();
+			let pos = available_names.iter().position(|x| &name == x);
+
+			if pos.is_none() {
+				return Err(format!(
+					"Invalid neighbour name: {:?}, available: {:?}",
+					name, available_names
+				));
 			}
+
+			let pos = pos.unwrap();
+
+			config.ops[pos] = score_parsed.unwrap();
 		},
-		Mode::Show => {
-			println!("Client config:");
-			println!("{:#?}", config);
+		Config::SecretKey => {
+			let sk_vec: Vec<String> = data.split(",").map(|x| x.to_string()).collect();
+			if sk_vec.len() != 2 {
+				return Err("Invalid secret key passed, expected 2 bs58 values separated by commas, e.g.:\n\"2L9bbXNEayuRMMbrWFynPtgkrXH1iBdfryRH9Soa8M67,9rBeBVtbN2MkHDTpeAouqkMWNFJC6Bxb6bXH9jUueWaF\"".to_string());
```

Comment:
Can we break this into several lines?
Its best to add `format_strings = true` inside `rustfmt.toml` file..

Reply:
Solved, added `format_strings = true` to `rustfmt.toml`